iphone/Classes/MediaModule.m

```diff
@@ -1863,7 +1863,8 @@ - (void)imagePickerController:(UIImagePickerController *)picker_ didFinishPickin
             }
             
             if (resultImage == nil) {
-                resultImage = (editedImage != nil) ? editedImage : originalImage;
+                UIImage *tempImage = (editedImage != nil) ? editedImage : originalImage;
+                resultImage = [TiUtils adjustRoation:tempImage];
```

Comment:
This method is misspelled (`adjustRoation ` instead of `adjustRotation`). I'm wondering how it could have worked in the tests.